hr_holidays_legal_leave/models/hr_holidays_status.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models, api
+
+
+class HolidaysType(models.Model):
+    _inherit = "hr.holidays.status"
+
+    is_annual = fields.Boolean(
```

Comment:
Yes, please. I think it's a good change. We can cherry-pick the commit directly in v10 when done in v9.